dts/bindings/base/base.yaml

```
@@ -17,93 +26,188 @@ properties:
   compatible:
     type: string-array
     required: true
-    description: compatible strings
+    description: |
+      This property is a list of strings that essentially define what
+      type of hardware or other resource this devicetree node
+      represents. Each device driver checks for specific compatible
+      property values to find the devicetree nodes that represent
+      resources that the driver should manage.
+
+      The recommended format is "vendor,device", The "vendor" part is
+      an abbreviated name of the vendor. The "device" is usually from
+      the datasheet.
+
+      The compatible property can have multiple values, usually ordered from
```

Not usually, always.

```suggestion
      The compatible property can have multiple values, ordered from
```